packages/gatsby-source-contentful/src/hooks.js

```diff
@@ -0,0 +1,17 @@
+// @ts-check
+import { getImageData } from "gatsby-plugin-image"
+
+import { createUrl } from "./image-helpers"
+
+export function useContentfulImage({ image, ...props }) {
+  return getImageData({
```

Comment:
This seems like it's worth memoizing this with `useMemo`. Wont `getImageData` re-run on each render?